resource_customizations/numaflow.numaproj.io/InterStepBufferService/actions/discovery.lua

```diff
@@ -0,0 +1,18 @@
+local actions = {}
+actions["force-promote"] = {
+  ["disabled"] = true,
+  ["iconClass"] = "fa-solid fa-fw fa-forward"
+}
+
+-- force-promote
+local forcePromote = false
+if (obj.metadata.ownerReferences ~= nil and obj.metadata.ownerReferences[1] ~= nil and obj.metadata.ownerReferences[0].kind == "ISBServiceRollout") then
+  forcePromote = true
+end
+if (obj.metadata.labels ~= nil and obj.metadata.labels["numaplane.numaproj.io/force-promote"] == "true") then
```

Comment:
for this and the others, can we only show it if the "upgrade-state" label is set to "in-progress"?

Reply:
Can do!

Reply:
looks good now, thanks